src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -1294,6 +1349,18 @@ def test_process_save_attribute_declarations(parser):
     processor.process_declarations(fake_parent, fparser2spec.children, [])
     var4 = fake_parent.symbol_table.lookup("var4")
     assert var4.datatype.intrinsic == ScalarType.Intrinsic.INTEGER
+    assert isinstance(fake_parent.symbol_table.lookup("var4").interface,
+                      StaticInterface)
+
+    # Test when is part of an UnknownDataType (target attribute in this case)
+    # it becomes and UnknownInterface
```

Comment:
s/and/an/ again, does it have to be an `UnknownInterface` given that we know where the declaration is?

Reply:
It is not enough to know where the declaration is, attributes in the declaration can change the interface, and for UnknownDataType we say that there are attributes that we don't understand.